docs/guide/architecture.md

```diff
@@ -1,15 +1,31 @@
 # Architecture
 
-![Architecture](img/ez5_architecture.png "Architecture")
+A motto for eZ architecture is to **use APIs** that will be maintained in the long term to **ease upgrades and provide lossless couplings** between all parts of the architecture, at the same time improving the migration capabilities of the system.
 
-The architecture of eZ Platform is layered and uses clearly defined APIs between the layers.
-
-- The **business logic** is defined in the kernel. This business logic is exposed to applications via an API (the [Public API](../api/public_php_api.md)). Developers rely on this to develop websites and web applications using Symfony to organize the way they develop the user interface layer.
+The structure of eZ Platform bundles is based on the Symfony framework
```

Comment:
```suggestion
The structure of an eZ Platform app is based on the Symfony framework
```
Isn't this architecture on a higher level than bundles? To me the sentence implies the internal structure of a bundle